### PR TITLE
Use only unsigned value for array access

### PR DESCRIPTION
**Pull request by henrygab**
GCC catches this bug.

```
...\libraries\Adafruit_TinyUSB_Library\src\Adafruit_USBD_HID.cpp:
      Line 181 Char 22
      In member function 'bool Adafruit_USBD_HID::keyboardPress(uint8_t, char)':
      warning: array subscript has type 'char' [-Wchar-subscripts]
  181 |   if (_ascii2keycode[ch][0])
      |                      ^~
...\libraries\Adafruit_TinyUSB_Library\src\Adafruit_USBD_HID.cpp:
      Line 183 Char 31
      warning: array subscript has type 'char' [-Wchar-subscripts]
  183 |   keycode[0] = _ascii2keycode[ch][1];
      |                               ^~
```

### CODE REVIEW

**Comment by ladyada** [comment]
please clang format and let us know when it passes CI :)

**Comment by hathach** [comment]
@henrygab clang is auto format code, it is mandatory for Adafruit arduino library, you can simply run it with 

```
$ clang-format -i *
```

you need to install it with `apt install clang` (not sure on windows/macos)

**Comment by henrygab** [comment]
@ladyada .. Ok, it's better now ..  CI passes.  Also, apologies for marking it ready before verifying CI success.

@hathach .. thanks for the pointer.

**Review by hathach — APPROVED** [review]
thank you for your PR. look great now.

**Comment by hathach** [comment]
> @ladyada .. Ok, it's better now .. CI passes. Also, apologies for marking it ready before verifying CI success.
> 
> @hathach .. thanks for the pointer.

no problem at all, actually there is no need to wait for ci passed to mark PR as ready. There is no definitive guide, but IMHO, PR should be marked as draft if contributor think it is still WIP, and need more time/commit to complete it. Though he/she want to have early feedback, conversation as well as pointer/reference in the big work instead of waiting until all the changes is complete (which the repo maintainer may have different view/taste etc ...).

Sum up, if you think the PR is complete and ready for a merge, just make it ready. CI failure is very common (by all sort of things).

**Pull request merged**
